src/batched/KokkosBatched_Copy_Decl.hpp

```diff
@@ -38,6 +38,32 @@ namespace KokkosBatched {
              const BViewType &B);
     };
 
+
+    ///
+    /// Selective Interface
+    ///
+    template<typename MemberType,
+             typename ArgTrans,
+             typename ArgMode>
+    struct Copy {
+      template<typename AViewType,
+               typename BViewType>
+      KOKKOS_FORCEINLINE_FUNCTION
+      static int
+      invoke(const MemberType &member,
+             const AViewType &A,
+             const BViewType &B) {
+        int r_val = 0;
+        if (std::is_same<ArgMode,Mode::Serial>::value) {
```

Comment:
I feel like this code would be cleaner with a tag to pick serial vs. team, than the current class template specialization design.  I've been trying tag-based design for my in-progress fix for https://github.com/trilinos/Trilinos/issues/4418 and it works really well.  That may be out of scope for this PR, but I think it's worth thinking about for the future.